pkg/admin/model/mock_rule.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model
+
+import (
+	"gorm.io/gorm"
+)
+
+type MockRule struct {
+	ID          uint   `json:"id"`
+	ServiceName string `json:"serviceName"`
+	MethodName  string `json:"methodName"`
+	Rule        string `json:"rule"`
+	Enable      bool   `json:"enable"`
+}
+
+func (m *MockRule) ToMockRuleEntity() *MockRuleEntity {
+	return &MockRuleEntity{
+		Model: gorm.Model{
+			ID: m.ID,
+		},
+		ServiceName: m.ServiceName,
+		MethodName:  m.MethodName,
+		Rule:        m.Rule,
+		Enable:      m.Enable,
+	}
+}
+
+type MockRuleEntity struct {
+	gorm.Model
+	ServiceName string `gorm:"type:varchar(255)"`
+	MethodName  string `gorm:"type:varchar(255)"`
+	Rule        string `gorm:"type:text"`
+	Enable      bool
```

Comment:
Is this definition the same as the original one listed here https://cn.dubbo.apache.org/zh-cn/overview/reference/admin/mock/#mock-rule?

We need to make sure previous users can smoothly migrate to the new version using the same backend MySQL server.

Reply:
Make sure to use the same schema

Reply:
Also, we should make sure new users know how to init the schema.

Reply:
I think GORM AutoMigrate can handle above possible compatibility issues. 
[GORM-AutoMigrate](https://gorm.io/zh_CN/docs/migration.html#AutoMigrate)